cpp/src/ray/util/process_helper.cc

```diff
@@ -100,13 +83,14 @@ void ProcessHelper::RayStart(CoreWorkerOptions::TaskExecutionCallback callback)
     }
   }
 
+  std::unique_ptr<ray::gcs::GlobalStateAccessor> global_state_accessor = nullptr;
```

Comment:
Why we don't initial GlobalStateAccessor here?

Reply:
We don't always need this accessor. For example if session_dir is passed in command line in worker, we don't need it.